subsquid/db/migrations/1662371228302-Init.js

```diff
@@ -0,0 +1,105 @@
+module.exports = class Init1662371228302 {
+  name = "Init1662371228302";
+
+  async up(db) {
+    await db.query(
```

Comment:
This is an auto-generated file so I don't think I have much power here. If I change it, it will need to be manually done every time the schema changes. Anyway, this is just about the creation of the tables so it should only run once and with no data right?

Reply:
This is the current ER diagram:

![er-diagram](https://user-images.githubusercontent.com/6484966/188581482-9c6a59d0-d53f-4de6-af3e-a56a70ab1370.png)

Reply:
This is a great convo. in the latest Firesquid implementation, we've generated everything from scratch and migrated (inside docker). 

But this approach is certainly significant when it comes to staging/production. 

I couldn't find the deploy script for this, but we will lose data if we generate the migration file from scratch every time we release it.

Reply:
![Screen Shot 2022-09-06 at 17 05 16](https://user-images.githubusercontent.com/6484966/188670097-af632091-36d4-4de2-a268-311cdf5547a3.png)